samza-core/src/main/scala/org/apache/samza/util/CoordinatorStreamUtil.scala

```diff
@@ -48,6 +48,23 @@ object CoordinatorStreamUtil extends Logging {
     new MapConfig(coordinatorSystemConfig)
   }
 
+  /**
+   * Creates coordinator stream from config.
```

Comment:
maybe we need to move the comment inside` // ..if it doesnt exist` to the java doc of this method. I feel that would declare better the nature of this method.

Reply:
Updated